### PR TITLE
LRQA-66271 Write TranslateLocalizable test

### PR DESCRIPTION
**Pull request by dereckalexander**
[LRQA-66271](https://issues.liferay.com/browse/LRQA-66271) Automate Test Scenario 28.05 of Data Engine Test Map

### CODE REVIEW

**Comment by dereckalexander** [comment]
ci:test:sf

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "Test Site Name");
+		}
+	}
+
+	@description = "This test specific validates that a Color field can be added to the builder."
```
Comment (emanuelfisilva) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase]: Please change the description accordingly.

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+		DataEngine.addField(
+			fieldFieldLabel = "Numeric",
+			fieldName = "Numeric");
+
+		DataEngine.addFieldNested(
+			fieldFieldLabel = "Date",
+			fieldName = "Date",
+			targetFieldLabel = "Text");
+
+		WebContentStructures.saveCP(structureName = "Structure Name");
```
Comment (emanuelfisilva) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase]: For standardization, please change structureName for "_WC Structure Title_".

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+		NavItem.click(navItem = "Web Content");
+
+		WebContentNavigator.gotoAddWithStructureCP(structureName = "Structure Name");
```
Comment (emanuelfisilva) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase]: For standardization, please change structureName for "WC Structure Title".

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+		PortletEntry.publish();
+
+		WebContentNavigator.gotoEditCP(webContentTitle = "Web Content Title");
+
+		WebContent.viewWithStructureCP(
+			predefinedValue = "true",
+			webContentDate = "03/30/2021",
+			webContentText = "This is a text field");
+
+		WebContent.viewWithStructureCP(
```
Comment (emanuelfisilva) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase]: Please merge this validations into previous macro usage.

Comment (emanuelfisilva) [reply modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase]: Example: LocalizedNestedFieldsWebContent

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+			predefinedValue = "true",
+			webContentDate = "30/03/2021",
+			webContentText = "Este é um campo de texto");
+
+		WebContent.viewWithStructureCP(
+			predefinedValue = "true",
+			webContentNumeric = "456");
+
```
Comment (emanuelfisilva) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase]: As discussed, your test finishes here.

**Review by emanuelfisilva — CHANGES_REQUESTED** [review]

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase (hunk 1 above)
Comment (emanuelfisilva) [reply modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/Localizable.testcase]: Suggestion "_This test ensures that is possible to translate fields with Localizable set as true._"

**Review by emanuelfisilva — COMMENTED** [review]
LGTM

**Review by emanuelfisilva — APPROVED** [review]

**Comment by dereckalexander** [comment]
ci:forward

**Pull request closed without merging**